src/schema/rules/dataset_metadata.yaml

```diff
@@ -0,0 +1,40 @@
+dataset_description:
+    path: /dataset_description.json
+    fields:
+        Name: REQUIRED
+        BIDSVersion: REQUIRED
+        DatasetType: RECOMMENDED
+        License: RECOMMENDED
+        Authors: OPTIONAL
+        Acknowledgements: OPTIONAL
+        HowToAcknowledge: OPTIONAL
+        Funding: OPTIONAL
+        EthicsApprovals: OPTIONAL
+        ReferencesAndLinks: OPTIONAL
+        DatasetDOI: OPTIONAL
+        GeneratedBy: RECOMMENDED
+        SourceDatasets: RECOMMENDED
+
+derivative_description:
+    path: /dataset_description.json
+    selectors:
+        dataset.DatasetType == "derivative"
```

Comment:
How do you envision the `dataset` variable being structured? What would be available in it?

Reply:
Right. Unclear. I was assuming that it would basically be the `dataset_description.json` object, or a semantically useful subset of it, a la @rwblair's nifti_header subset.

Reply:
I think we also have used it for lists of modalities present in the dataset in other PRs.